catalog/confluence/catalog-confluence-source/src/main/java/org/codice/ddf/confluence/source/ConfluenceSource.java

```diff
@@ -330,6 +337,10 @@ public void setAdditionalAttributes(List<String> attributes) {
         additionalMetacardAttributes = Permissions.parsePermissionsFromString(attributes);
     }
 
+    public void setBodyExpansion(String bodyExpansion) {
+        this.bodyExpansion = bodyExpansion;
```

Comment:
The only vulnerability I can think of here would be to supply really large strings as input to use up memory. This is protected under the admin context right now so I am not real worried about it. Let me know if you think there is a different attack vector that needs to be considered or if you feel the size should be checked.